extension/mediator.js

```diff
@@ -27,7 +32,16 @@ class Mediator {
     }
 
     init() {
-        browser.runtime.sendMessage({ command: "monitorTabLoad" });
+        this.telemetry.loadSchema()
+            .then(() => {
+                browser.runtime.sendMessage({ command: "monitorTabLoad" })
```

Comment:
sometimes mediator.start is called before telemetry schema is loaded on page refresh. I don't understand why this happens, probably old subscription to "responseMonitorTabLoad" in background script survives the refresh.